modules/lvms-creating-logical-volume-manager-cluster.adoc

```diff
@@ -74,6 +74,15 @@ spec:
 If you add or remove a `deviceClass`, then the update reflects in the cluster only after deleting and recreating the `topolvm-node` pod.
 Configure the local device paths of the disks as an array of values in the `deviceSelector` field.
 When configuring multiple device classes, you must specify the device path for each device.
++
+[WARNING]
+====
+It is recommended to avoid referencing disks using symbolic naming, such as `/dev/sdX`, as these names may change across reboots within RHCOS. Instead, you must use stable naming schemes, such as `/dev/disk/by-path/` or `/dev/disk/by-id/`, to ensure consistent disk identification.
+
+With this change, you might need to adjust existing automation workflows in the cases where monitoring collects information about the install device for each node.
+
+For more information, see the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_file_systems/assembly_overview-of-persistent-naming-attributes_managing-file-systems[{op-system-base} documentation].
+====
```

Comment:
This is one way to try and fix the indentation issue, but double check on your own local build to see if it works:
```suggestion
+
--
[WARNING]
====
It is recommended to avoid referencing disks using symbolic naming, such as `/dev/sdX`, as these names may change across reboots within RHCOS. Instead, you must use stable naming schemes, such as `/dev/disk/by-path/` or `/dev/disk/by-id/`, to ensure consistent disk identification.

With this change, you might need to adjust existing automation workflows in the cases where monitoring collects information about the install device for each node.

For more information, see the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_file_systems/assembly_overview-of-persistent-naming-attributes_managing-file-systems[{op-system-base} documentation].
====
--
```